src/prompt.ts

```diff
@@ -0,0 +1,36 @@
+import * as readline from "readline";
+import { messages, PLACEHOLDER } from "./config/messages.js";
+import { getPlayerTokenChar, boardLayout } from "./layout/cliLayout.js";
+import { GameState, PlayerNum } from "./types/gameState.js";
+const rl: readline.Interface = readline.createInterface({
```

Comment:
```suggestion

const rl: readline.Interface = readline.createInterface({
```

Sorry for my psychorigidity again

Reply:
I'm too ... I understand you ...